resources/js/components/shared/TimelinePlot.vue

```diff
@@ -0,0 +1,393 @@
+<template>
+  <div id="timeline_plot"></div>
```

Comment:
What if there are multiple timeline plots on the page?  The ID attribute is supposed to refer to a single unique element on the page.  Although multiple IDs is technically undefined behavior, many browsers will handle it gracefully, but we don't want to rely upon that.

Reply:
That's a really good point, but at the same time d3 needs to target some input `div` in which to construct the svg object. Do you have other suggestions for making this work?